stubs/runtime.php

```diff
@@ -37,7 +35,7 @@
     MaintenanceMode::setUp();
 
     $kernel = $app->make(ConsoleKernel::class);
-    $status = $kernel->handle($input = new ArgvInput, new ConsoleOutput);
+    $status = $kernel->handle($input = new ArgvInput(), new ConsoleOutput());
```

Comment:
Is this PSR-12?

Reply:
Run `composer exec -- phpcs --standard=PSR12 -s src/ stubs/`

Reply:
I hate PHPCS. How TF can I make it use a `phpcs.xml.dist`?

```
./vendor/bin/phpcs --config-show
Using config file:
````

Reply:
Simply create that file in the root and the contents,
e.g. https://github.com/szepeviktor/phpcs-psr-12-neutron-hybrid-ruleset/blob/master/PSR12NeutronRuleset/ruleset.xml

Reply:
Oh no! :)
https://github.com/squizlabs/PHP_CodeSniffer/issues/2862#issuecomment-583890204